modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Chart.js

```diff
@@ -140,17 +147,36 @@ export default function Chart({
 
 	const [, addHistoricalWarning] = useHistoricalWarning();
 
-	const {actions, state: chartState} = useChartState({
-		publishDate,
-		timeSpanKey,
-	});
+	const chartState = useChartState();
+
+	const addDataSetItems = useAddDataSetItems();
+
+	const setLoading = useSetLoading();
+
+	const changeTimeSpanKey = useChangeTimeSpanKey();
+
+	const previousTimeSpan = usePreviousTimeSpan();
+
+	const nextTimeSpan = useNextTimeSpan();
+
+	const isPreviousPeriodButtonDisabled = useIsPreviousPeriodButtonDisabled();
```

Comment:
Cosmetic thing: Brian might ask for these to be sorted (although he may not know that they are side-effect-free, so perhaps he'll say nothing).

Reply:
I separated between variables and functions and sort each group. Let's see.